client/landing/gutenboarding/components/link/index.tsx

```diff
@@ -5,7 +5,6 @@ import { Button } from '@wordpress/components';
 import { Link as RouterLink, LinkProps } from 'react-router-dom';
 import React, { forwardRef, FunctionComponent } from 'react';
 import { Assign } from 'utility-types';
-
```

Comment:
Let's also revert this small whitespace change, so this file is no longer touched in this diff :slightly_smiling_face:

Reply:
Fixed. https://github.com/Automattic/wp-calypso/pull/39312/commits/f3ca829a3730ef2832a1a240aca921683b63ae59